cypress/e2e/state-calculator.cy.ts

```diff
@@ -88,54 +84,6 @@ describe('rewiring-america-state-calculator', () => {
       });
   });
 
-  it('shows an empty state if you are not eligible for any incentives for your chosen project', () => {
```

Comment:
I'm actually not sure it's even possible to get this state anymore? Need to think about it. If it's not, there's even more code I can delete.